CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 Description of the upcoming release here.
 
+## [Hotfix 0.35.4]
```

Comment:
I just realized I made this up without thinking about it. Should this be `Version` still?